### PR TITLE
Documented S3 as a source

### PR DESCRIPTION
**Pull request by stepgerner**
Additionally added new h2 for preview features that do not require enablement.

https://jira.wandisco.com/browse/DOCU-1183

### CODE REVIEW

File: docs/preview-features.md [hunk 1]
```diff
@@ -43,3 +43,37 @@ Enable this feature with the following property:
...
+Supply the `--source` parameter to the [`filesystem add s3a` command](./command-reference.md#filesystem-add-s3a).
+
+```text title="Example"
+filesystem add s3a --file-system-id mytarget --bucket-name mybucket1 --credentials-provider org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider --access-key B6ZAI18Z3UIO002Y777A --secret-key OP87Chokisf4hsTP0Q5j95yI904lT7AaDBGJpp0D --source
+```
+
+### Databricks Hive agent
+
+LiveData Migrator supports metadata migration to Databricks Delta Lake.
+
```
Comment (benradstone) [inline docs/preview-features.md:70]: Line 70: can we include a *Creating a Databricks agent through the UI* subsection here?

File: docs/preview-features.md [hunk 2]
```diff
@@ -43,3 +43,37 @@ Enable this feature with the following property:
...
+#### Creating an Amazon S3 source in the CLI
+
+Supply the `--source` parameter to the [`filesystem add s3a` command](./command-reference.md#filesystem-add-s3a).
+
+```text title="Example"
+filesystem add s3a --file-system-id mytarget --bucket-name mybucket1 --credentials-provider org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider --access-key B6ZAI18Z3UIO002Y777A --secret-key OP87Chokisf4hsTP0Q5j95yI904lT7AaDBGJpp0D --source
+```
+
+### Databricks Hive agent
```
Comment (benradstone) [inline docs/preview-features.md:67]: Line 67: Change this to *Databricks metadata agent*

File: docs/preview-features.md [hunk 3]
```diff
@@ -43,3 +43,37 @@ Enable this feature with the following property:
...
+
+LiveData Migrator supports metadata migration to Databricks Delta Lake.
+
+#### Creating a Databricks hive agent through the CLI
```
Comment (benradstone) [inline docs/preview-features.md:71]: Line 71: Change this to **Databricks metadata agent**

File: docs/preview-features.md (hunk 1 above)
Comment (stepgerner) [reply docs/preview-features.md:70]: I failed to spot the Ui instructions, I found them now. Thank you, will do!

**Review by benradstone — APPROVED** [review]

**Pull request merged**